integration_tests/src/main/python/conditionals_test.py

```diff
@@ -21,7 +21,15 @@
 import pyspark.sql.functions as f
 
 all_gens = all_gen + [NullGen()]
-all_nested_gens = array_gens_sample + struct_gens_sample
+
+# map_gens_sample have one nested map(map) that cudf will cause bug, will file one bug
```

Comment:
Better to add the link after filing the bug.

Reply:
```suggestion
# "map_gens_sample" has one nested "map(map)" gen that will lead to a bug in cudf as below. (tracked by <issue url>)
<error stack log>
```

Reply:
The bug has been fixed in branch-21.10, no need to report one.